docker/check_fast_version.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+try:
+    from gensim.models.word2vec_inner import FAST_VERSION
+
```

Comment:
It does run from [here](https://github.com/parulsethi/gensim/blob/19ce3ca20d77dfe375b0d23353971fb6637e3319/docker/Dockerfile#L136-L137) in dockerfile with both python 2 and 3. And only the pinned versions of numpy/scipy/gensim installed in the container are used